plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/prior_box.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/prior_box.hpp"
+
+using LayerTestsDefinitions::PriorBoxLayerTest;
+
+namespace {
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+    // InferenceEngine::Precision::FP16,
+};
+
+// The count of tests will be too large for CI, close most checks here
+const std::vector<std::vector<float>> minSizes = {{256.0f}};
+const std::vector<std::vector<float>> maxSizes = {{315.0f}};
+const std::vector<std::vector<float>> aspectRatios = {
+    {1.0f},
+    // {2.0f, 3.0f}
+};
+const std::vector<std::vector<float>> density = {{2.0f, 4.0f}};
+const std::vector<std::vector<float>> fixedRatios = {{2.0f, 4.0f}};
+const std::vector<std::vector<float>> fixedSizes = {{2.0f, 4.0f}};
+const std::vector<bool> clip = {
+    false,
+    // true
+};
+const std::vector<bool> flip = {
+    false,
+    // true
+};
+const std::vector<float> steps = {
+    1.0f,
+    // 0.0f,
+    // -1.0f
+};
+const std::vector<float> offsets = {
+    0.0f,
+    // 0.5f
+};
+const std::vector<std::vector<float>> variances = {
+    {0.1f},
+    // {0.1f, 0.2f, 0.3f, 0.4f}
+};
+const std::vector<bool> scaleAllSizes = {false, true};
+// some attributes are complict with others, use this to switch
```

Comment:
What does this comment mean?

Reply:
when fixedSize is set, minSize and maxSize shall not be set, otherwise ngraph implementation will produce wrong results which has different output shape requested by spec,  fixed_ratio and aspectRatio have the same issue.  So need a switch to do this and i check this in setUp now

Reply:
the check is done in https://github.com/Flex-plaidml-team/openvino/pull/14/files#diff-e9a45f615029f9d7c89f453c5a9be69fdf52e85ca29256cf1217d037654bb931R97

Reply:
I see, maybe it is better to reword the comments to something like "The following added bool vectors are used to mask corresponding attributes. "

Reply:
Done